app/components/input_field_component/input_field_component.html.haml

```diff
@@ -0,0 +1,45 @@
+%div
+    - unless @label == ""
+        .text-input-label
+            = @label
+
+    - has_error = @error_message != ""
+
+    - case @type
+    - when "select"

```

Comment:
so here instead createe, 3 components  `selectInput` (already exist) , `textareaInput`, and `textInput`